js_modules/dagster-ui/packages/ui-core/src/sensors/SensorDetails.tsx

```diff
@@ -152,6 +150,22 @@ export const SensorDetails = ({
               )}
             </td>
           </tr>
+          {sensor.nextTick && daemonHealth && running && (
+            <tr>
+              <td>Next tick</td>
+              <td>
+                <Box
+                  flex={{direction: 'row', gap: 8, alignItems: 'center'}}
+                  style={{marginTop: '-2px'}}
+                >
```

Comment:
Are the flex props needed? There's only the one child, so It doesn't seem like it, but perhaps there's something I'm missing.

Also, is the margin-top needed? In the screenshot it looks like it drags the timestamp string a bit too high.

I think these values are present on the other row because there's a tag adjacent -- in this case, I think you can just render the `TimestampDisplay` with no extra wrappers.